lighthouse-core/report/html/templates.html

```diff
@@ -497,6 +497,32 @@
         animation-delay: 250ms;
     }
 
+    /* The plugin badge overlay */
```

Comment:
this comment might make more sense below this, above `.lh-gauge__wrapper--plugin`, since *all* category gauges will have an `svg-wrapper`, they just won't have the `:before` content

Reply:
done